torch/utils/module_tracker.py

```diff
@@ -76,6 +77,74 @@ def is_bw(self):
         """
         return torch._C._current_graph_task_id() != -1
 
+    def get_known_fqn(self, mod):
+        """
+        Return the fqn for the given module if it is known to the Module Tracker, otherwise None.
+        """
+        return self._known_modules.get(mod, None)
+
+    def register_user_hooks(
+        self,
+        pre_fw_hook: Optional[Callable] = None,
+        post_fw_hook: Optional[Callable] = None,
+        pre_bw_hook: Optional[Callable] = None,
+        post_bw_hook: Optional[Callable] = None,
+    ):
+        """
+        Registers user-specified hooks to be called before/after the forward/backward pass for each
```

Comment:
We should mention here that when the hooks called, the current module is always inside the parents set

Reply:
I have already mentioned that in 

> Note:
            If the module is not alive during the backward pass, the pre_bw_hook and post_bw_hook will
            will receive None as the module argument.
            The module fqn will be present in the ``parents`` attribute when each of the hooks is called.
            Hooks are intended to be used as markers only not to modify the inputs/outputs.

Reply:
Ho my bad, sounds good!